.github/workflows/test.yaml

```diff
@@ -1,13 +1,18 @@
 name: test
 on:
   pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
Those are default types, we don't need to list them:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request